### ISSUE TITLE
An unhandled exception occurred: Cannot read property 'cli' of undefined

### ISSUE THREAD

**Issue by amyloula** [flowaccount/nx-plugins#17]
https://github.com/flowaccount/nx-plugins/blob/df7b74f4728ed356f6c8cc81150ded3c19234bd2/libs/nx-serverless/src/utils/normalize.ts#

Running `nx serve <app-name>` or `nx build <app-name>`

throws a TypeError:

```
Serverless: getting all functions
An unhandled exception occurred: Cannot read property 'cli' of undefined
```

Log:

```
[error] TypeError: Cannot read property 'cli' of undefined
    at getEntryExtension (<projectroot>/libs/nx-serverless/src/utils/normalize.ts:166:21)
    at Object.exports.getEntryForFunction (<projectroot>/libs/nx-serverless/src/utils/normalize.ts:115:15)
    at <projectroot>/libs/nx-serverless/src/utils/normalize.ts:25:19
    at arrayEach (<projectroot>/node_modules/lodash/lodash.js:516:11)
    at Function.forEach (<projectroot>/node_modules/lodash/lodash.js:9342:14)
    at Object.assignEntriesToFunctionsFromServerless (<projectroot>/libs/nx-serverless/src/utils/normalize.ts:23:5)
    at MapSubscriber.project (<projectroot>/libs/nx-serverless/src/builders/build/build.impl.ts:38:14)
    at MapSubscriber._next (<projectroot>/node_modules/rxjs/src/internal/operators/map.ts:84:29)
    at MapSubscriber.Subscriber.next (<projectroot>/node_modules/rxjs/src/internal/Subscriber.ts:99:12)
    at SwitchMapSubscriber.notifyNext (<projectroot>/node_modules/rxjs/src/internal/operators/switchMap.ts:172:24)
    at InnerSubscriber._next (<projectroot>/node_modules/rxjs/src/internal/InnerSubscriber.ts:17:17)
    at InnerSubscriber.Subscriber.next (<projectroot>/node_modules/rxjs/src/internal/Subscriber.ts:99:12)
    at CombineLatestSubscriber.notifyNext (<projectroot>/node_modules/rxjs/src/internal/observable/combineLatest.ts:313:26)
    at InnerSubscriber._next (<projectroot>/node_modules/rxjs/src/internal/InnerSubscriber.ts:17:17)
    at InnerSubscriber.Subscriber.next (<projectroot>/node_modules/rxjs/src/internal/Subscriber.ts:99:12)
    at MergeMapSubscriber.notifyNext (<projectroot>/node_modules/rxjs/src/internal/operators/mergeMap.ts:168:22)
```

**Comment by wickstargazer** [comment]
Hi! I will look into it, i think its the serverless verions can you help post the version of thr serverless you are using?

**Comment by wickstargazer** [comment]
@amyloula I found the issue, it was trying to log `WARNING: More than one matching handlers found for '${fileName}'. Using '${_.first( sortedFiles)}'.` 

and it was referring to a wrong instance of serverless, i fixed it, it will be released in the next version.

Thankz for the report! Cheers

**Comment by amyloula** [comment]
That’s great, thanks very much!

**Comment by wickstargazer** [comment]
fixed in 0.5.0
